File: tests/integration/features/conditionals/conditionals.rb
# Ruby's nested elsif / else chain collapses to flat <else_if> / <else>
# siblings of <if> (post-walk collapse_else_if_chain). `elsif` is
# renamed to <else_if> (underscore per Principle #1).

def classify(n)
  if n < 0
    "neg"
  elsif n == 0
    "zero"
  elsif n < 10
    "small"
  else
    "big"
  end
end

def label(n)
  n > 0 ? "positive" : "non-positive"
end

File: tests/integration/features/conditionals/conditionals.rs
// Conditional shape: `else if` chain collapses to flat <else_if>
// siblings. Ternary via if-expression keeps <then>/<else>.

fn classify(n: i32) -> &'static str {
    if n < 0 {
        "neg"
    } else if n == 0 {
        "zero"
    } else if n < 10 {
        "small"
    } else {
        "big"
    }
}

fn label(n: i32) -> &'static str {
    if n > 0 { "positive" } else { "non-positive" }
}
